apisupport.refactoring/test/qa-functional/data/goldenfiles/org/netbeans/modules/apisupport/refactoring/RenameTest/whereUsed.ref

```diff
@@ -1,3 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 testrename.MyDataLoader
 Found 3 occurance(s).
 
```

Comment:
This is dangerous. And will cause unittests to fail. In this case it causes no problems, because the unittest referencing the file `RenameTest#testWhereUsed` is commented out. In that test a new file is created with some test output and that test output is literally compared to the golden file (`NbTestCase#assertFile`).  As there is no IP here, I'd add this file to the build.xml exclude list for rat (the *.pass files are already excluded by a wildcard).